cfme/tests/cli/test_appliance_console.py

```diff
@@ -67,6 +75,54 @@ def test_appliance_console_set_timezone(timezone, temp_appliance_preconfig_modsc
     temp_appliance_preconfig_modscope.appliance_console.timezone_check(timezone)
 
 
+def test_appliance_console_datetime(temp_appliance_preconfig_funcscope):
+    """Grab fresh appliance and set time and date through appliance_console and check result"""
+    app = temp_appliance_preconfig_funcscope
+    command_set = ('ap', '', '3', 'y', '2020-10-20', '09:58:00', 'y', '')
+    app.appliance_console.run_commands(command_set)
+
+    def date_changed():
+        return bool(app.ssh_client.run_command("date | grep 'Tue Oct 20 10:00'").success)
+
+    assert bool(app.ssh_client.run_command("date | grep 2020").success)
+    wait_for(date_changed)
+
+
+def test_appliance_console_db_maintenance_hourly(temp_appliance_time_crud):
+    """Test database hourly re-indexing through appliance console"""
+    app = temp_appliance_time_crud
+    command_set = ('ap', '', '7', 'y', 'n', '')
+    app.appliance_console.run_commands(command_set)
+
+    def maintenance_run():
+        return bool(app.ssh_client.run_command(
+            "grep REINDEX /var/www/miq/vmdb/log/hourly_continuous_pg_maint_stdout.log").success)
+
+    wait_for(maintenance_run, timeout=300)
+
+
+@pytest.mark.parametrize('period', [
+    ['hourly'],
+    ['daily', '10'],
+    ['weekly', '10', '2'],
+    ['monthly', '10', '20']
+], ids=['hour', 'day', 'week', 'month'])
+def test_appliance_console_db_maintenance_periodic(period, temp_appliance_time_crud):
+    """Tests full vacuums on database through appliance console"""
+    app = temp_appliance_time_crud
+    command_set = ['ap', '', '7', 'n', 'y']
+    command_set.extend(period)
+    app.appliance_console.run_commands(command_set)
+
+    def maintenance_run():
+        return bool(app.ssh_client.run_command(
```

Comment:
I don't think you need to wrap that in bool, since success returns a boolean value:

    @property
    def success(self):
        return self.rc == 0